### ISSUE TITLE
Add a README with installation instructions and usage

### ISSUE THREAD

**Issue by dit7ya** [WorldBrain/storex-hub-demo-memex-gist-sharer#1]
I have no idea how to set this up, or even what it is! (Is it a webextension? Or a local node cli?) As this is a demo app for the storexhub functionality (and there is no documentation for anything yet), it is absolutely necessary to have a README.

**Comment by ShishKabab** [comment]
Indeed, one of the next items on my list is to have a coherent approach to documentation. Just wanted some APIs to be stable before making an easy-to-follow tutorial for creating plugins for Storex Hub. Thanks for reminding me to also add this to a REAMDE in this repo when I've written this!

I've started some documentation here, but there's some changes coming up this week:
https://worldbrain.github.io/storex-docs/#/storex-hub/

**Comment by dit7ya** [comment]
Any update on this?

**Comment by ShishKabab** [comment]
Unfortunately, we've had to focus our efforts on other things (we're 2,5 developers), so Storex Hub is now paused until we discover more interest in it. I'm still excited about the idea of being able to easily integrate data from different apps and work with it in different ways, but right now we as a small team need to focus on something that is more coinnected with our immediate user needs (such as sharing notes with other people, annotatiing PDFs, etc.)

If there are people that really do want to build this out further, I'd be happy to support that. If not, I hope we can get back to this once we're a more mature and financially stronger organization.
